MBBSEmu.Tests/ExportedModules/Majorbbs/key_Tests.cs

```diff
@@ -99,6 +100,29 @@ public void haskey_Empty_Test()
 
         }
 
+        [Fact]
+        public void uidkey_Test()
```

Comment:
let's have other tests for
-  empty `accountKey`
- non-default key (i.e. insert a new key for the user, query specifically on it)
- unknown user with default key = true
- unknown user with custom key = false